static/app/views/alerts/incidentRules/ruleForm/index.tsx

```diff
@@ -795,7 +795,9 @@ const AlertListItem = styled(StyledListItem)`
 `;
 
 const ChartHeader = styled('div')`
-  padding: ${space(3)} ${space(3)} 0 ${space(3)};
+  position: relative;
+  bottom: -${space(1.5)};
```

Comment:
wouldn't this be a negative margin instead? i think bottom is intended for absolute position